nixos/modules/programs/steam.nix

```diff
@@ -68,6 +71,23 @@ in {
       '';
     };
 
+    extraCompatPackages = mkOption {
+      type = with types; listOf package;
+      default = [ ];
+      defaultText = literalExpression "[]";
+      example = literalExpression ''
+        with pkgs; [
+          luxtorpeda
+          proton-ge
+        ]
+      '';
+      description = mdDoc ''
```

Comment:
I'm not sure this is required anymore but there's tons of instances of this, so I guess it's fine to keep.

Reply:
do you mean that directly a string can be passed?

Reply:
Yeah, it's the default nowadays AFAIK.

Reply:
yeah okay that make sense to me then in general, as the module is quite consistent in still using mdDoc, i would like to keep it for the moment and leave it open for a refactoring, if that is okay from your side, otherwise i would remove it from every option to stay consistent.

Reply:
I think keeping it is fine, we can always remove this in another PR.